code/modules/vending/engivend.dm

```diff
@@ -4,21 +4,27 @@
 	icon_state = "engivend"
 	icon_deny = "engivend-deny"
 	req_access = list(ACCESS_ENGINE_EQUIP)
-	products = list(/obj/item/clothing/glasses/meson/engine = 2,
-					/obj/item/clothing/glasses/welding = 3,
-					/obj/item/multitool = 4,
+	products = list(/obj/item/clothing/glasses/meson/engine = 5,
+					/obj/item/clothing/glasses/welding = 5,
+					/obj/item/multitool = 5,
 					/obj/item/construction/rcd/loaded = 3,
 					/obj/item/grenade/chem_grenade/smart_metal_foam = 10,
-					/obj/item/geiger_counter = 5,
+					/obj/item/geiger_counter = 6,
 					/obj/item/stock_parts/cell/high = 10,
-          /obj/item/electronics/airlock = 10, 
+    		        /obj/item/electronics/airlock = 10, 
 					/obj/item/electronics/apc = 10, 
 					/obj/item/electronics/airalarm = 10,
 					/obj/item/electronics/firealarm = 10,
-					/obj/item/electronics/firelock = 10
+					/obj/item/electronics/firelock = 10,
+					/obj/item/rcd_ammo = 3
 					)
-	contraband = list(/obj/item/stock_parts/cell/potato = 3)
+	contraband = list(/obj/item/stock_parts/cell/potato = 3,
+					  /obj/item/stock_parts/cell/emproof = 5,
```

Comment:
No.

Reply:
"No." is not freed back
Reason for it was do to it having shitty stats and its a joke item.
/obj/item/stock_parts/cell/emproof
	name = "\improper EMP-proof cell"
	desc = "An EMP-proof cell."
	maxcharge = 500
	rating = 3

Also tube lights take 30 or 60 wats of power per tick so it would only be able to power a single light for legit 10~ seconds